client/state/ui/selectors/app-banner-is-visible.js

```diff
@@ -1,7 +1,12 @@
 import 'calypso/state/ui/init';
+import { getCurrentFlowName } from 'calypso/state/signup/flow/selectors';
 import { getCurrentLayoutFocus } from 'calypso/state/ui/layout-focus/selectors';
 
 export default function isAppBannerEnabled( state ) {
+	// Never show the AppBanner in the site setup flow.
+	if ( getCurrentFlowName( state ) === 'setup-site' ) {
+		return false;
+	}
```

Comment:
Hi @enejb, should we check here for the current flow name to be 'setup-site' and at the same time for the previous flow name to be 'onboarding' ?